File: IntellimentTutorial/IntellimentTutorial/Tab1/ModalView/ModalViewController.m
//
//  ModalViewController.m
//  IntellimentTutorial
//
//  Created by Tamboli, Ashfaque on 02/04/16.
//  Copyright © 2016 Tamboli, Ashfaque. All rights reserved.
//

#import "ModalViewController.h"

@interface ModalViewController ()

@end

@implementation ModalViewController
@synthesize  lblCandidateName;
@synthesize lblDesignation;
@synthesize lblEmail;
@synthesize lblPhoneNo;
- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    [self userDetails];
    
    
}

-(void) userDetails
{
    [self.lblCandidateName setText:@"Ashfaque Tamboli"];
    [self.lblDesignation   setText:@"Team Lead"];
    [self.lblEmail setText:@"ashfaque21@gmail.com"];
    [self.lblPhoneNo setText:@"09762478745"];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


- (IBAction)cancelAction:(id)sender
{
    [[self presentingViewController] dismissViewControllerAnimated:YES completion:nil];
}

@end

File: IntellimentSwiftApp/IntellimentSwiftApp/Tab1/ModalView/ModalViewController.swift
//
//  ModalViewController.swift
//  IntellimentSwiftApp
//
//  Created by Tamboli, Ashfaque on 06/04/16.
//  Copyright © 2016 Tamboli, Ashfaque. All rights reserved.
//

import Foundation
import UIKit

let candidateName = "Ashfaque Tamboli"
let designation = "Team Lead"
let emailId = "ashfaque21@gmail.com"
let phoneNo = "09762478745"

class ModalViewController: UIViewController
{
    @IBOutlet weak var lblCandidateName: UILabel!
    @IBOutlet weak var lblPhoneNo: UILabel!
    @IBOutlet weak var lblEmail: UILabel!
    @IBOutlet weak var lblDesignation: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        self.userDetails()
    }
    
    func userDetails()
    {
        lblCandidateName.text = candidateName
        lblDesignation.text = designation
        lblEmail.text = emailId
        lblPhoneNo.text = phoneNo
    }
    
    @IBAction func cancelAction(sender: AnyObject)
    {
        self.presentingViewController?.dismissViewControllerAnimated(true, completion: nil)
    }
    
}